study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/translation/TranslationsEnricher.java

```diff
@@ -134,7 +134,7 @@ private static void addVariablesToTemplate(
             List<Translation> translations = detectTranslationsForTemplateVariable(null, v, allTranslations);
 
             // add variables to template only if feature 'saveTranslationsToDbJson' is disabled
```

Comment:
Do we need template variables if translations are converted (or in new format) and already saved in the database ?
If we do:
This comment need to be updated to reflect the change made.